### ISSUE TITLE
Some versions of phpdbg are ignoring the __debugInfo magic method

### ISSUE THREAD

**Issue by ezzatron** [shivammathur/setup-php#93]
**Describe the bug**

For some PHP versions, the `phpdbg` executable provided by this action is ignoring the `__debugInfo` magic method.

**To Reproduce**

I've created a repo to reproduce the issue: [ezzatron/setup-php-action-repro](https://github.com/ezzatron/setup-php-action-repro)

But the gist of it is this PHPUnit test, which passes on equivalent Travis CI builds, (and weirdly, only under PHP 7.3 on GitHub Actions?):

```php
<?php

use PHPUnit\Framework\TestCase;

class ReproTest extends TestCase
{
    public function testDebugMagicMethod()
    {
        ob_start();
        var_dump(new ClassA());

        $this->assertRegExp('/\["a"]=>\s+string\(1\) "b"/', ob_get_clean());
    }
}

class ClassA
{
    public function __debugInfo() {
        return ['a' => 'b'];
    }
}
```

**Expected behavior**

The tests should pass, like this Travis build: https://travis-ci.org/ezzatron/setup-php-action-repro/builds/618399139

At the moment, they fail like this GitHub Actions build: https://github.com/ezzatron/setup-php-action-repro/commit/cf465a6517a7fd06811997914fe54807e045e235/checks?check_suite_id=333972681

**Screenshots**

<img width="1002" alt="Screen Shot 2019-11-29 at 9 56 55 am" src="https://user-images.githubusercontent.com/100152/69835605-9dc7e280-128e-11ea-9096-19450174c310.png">
<img width="1013" alt="Screen Shot 2019-11-29 at 9 56 38 am" src="https://user-images.githubusercontent.com/100152/69835606-9e607900-128e-11ea-965a-7234f16bdfe4.png">

**Additional context**

I'm the author of [Phony](https://eloquent-software.com/phony/), a PHP mocking library. I use the `__debugInfo()` magic method so that if a developer uses `var_dump()` on one of Phony's mocks (or something containing a mock), it doesn't produce thousands of lines of output about internal Phony classes. I discovered this bug because I have a test for this behaviour in Phony's test suite.

**Comment by shivammathur** [comment]
@ezzatron Thanks for reporting this.
Fixed in #94

**Comment by ezzatron** [comment]
No worries. Thanks for the speedy fix!
